src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(String color, double radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.PI * radius * radius;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: circle, area: " + getArea() + " sq. units, radius: " + radius
+                + " units, color: " + color);

```

Comment:
The `color` field is accessed directly in the `draw` method. Since `color` is a protected field in the `Figure` class, this is technically allowed, but it's generally better practice to use a getter method if one is available or to ensure that direct access is intentional and justified.